public/index.html

```diff
@@ -29,16 +40,27 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-	<!-- index.normal-->
-	<title>Sosialhjelp</title>
-</head>
+        <!-- index.normal-->
+        <link
+            href="https://dekoratoren.dev.nav.no/css/client.css"
```

Comment:
ENV variabler blir erstattet med statiske verdier når appen bygges. Det er derfor ikke mulig å hente miljøspesifikke variabler fra eks. Nais. Vi må finne en annen måte å kunne sette riktig URL ved bygging for test / prod.

Reply:
Dette ble løst med å ta i bruk express i stedet for å servere de statiske sidene med nginx.